cylc/flow/scripts/workflow_state.py

```diff
@@ -20,10 +20,13 @@
 
 Check or poll a workflow database for task statuses or completed outputs.
 
-The ID argument can target a workflow, or a cycle point, or a specific
-task, with an optional selector on cycle or task to match task status,
-output trigger (if not a status, or with --trigger) or output message
-(with --message). All matching results will be printed.
+The ID argument can target a workflow, or a cycle point, or a specific task,
+with an optional selector on cycle or task to match final task statuses,
+output trigger (with --triggers), or output message (with --messages).
+You cannot poll for transient states such as "submitted" and "running".
+Poll for the corresponding output triggers instead ("submitted", "started").
```

Comment:
Corrections:
- no longer accepts transient statuses, only final statuses
- no longer falls back to trigger if not a known status
- `--message` was wrong (it's `--messages`)